stress_load/helpers/topic.js

```diff
@@ -0,0 +1,8 @@
+const crypto = require("crypto");
+const encUtils = require("enc-utils");
+
+function generateRandomTopic() {
```

Comment:
how about using [generateRandomBytes32](https://github.com/WalletConnect/walletconnect-monorepo/blob/d6f5fb985084f89304315b6f091ae5db831c63bc/packages/utils/src/crypto.ts#L13) ?

Reply:
can't import from ES6 module into CommonJS in artillery

Reply:
That explains things